go/pkg/cas/client.go

```diff
@@ -89,6 +96,12 @@ func DefaultClientConfig() ClientConfig {
 
 		FindMissingBlobsBatchSize: 1000,
 
+		BatchUpdateBlobsConcurrency: 256, // arbitrary
+
+		// This is a suggested approximate limit based on current RBE implementation.
```

Comment:
this was copied from https://github.com/bazelbuild/remote-apis-sdks/blob/2dacc28301fb5eb8af27adf40456cac474f6f0e6/go/pkg/client/client.go#L122.

I prefer not link that line because that code is likely to be replaced with this new code.

Also I suspect the limit is somewhere in RBE server impl, which is closed-source, so I guess that's why that line isn't referring to anything either.

Reply:
Ack.